src/lambda/wizards/samInitWizard.ts

```diff
@@ -66,7 +76,14 @@ class DefaultCreateNewSamAppWizardContext implements CreateNewSamAppWizardContex
         })
 
         const choices = await picker.promptUser({
-            picker: quickPick
+            picker: quickPick,
+            onDidTriggerButton: (button, resolve, reject) => {
+                if (button === vscode.QuickInputButtons.Back) {
+                    resolve(undefined)
+                } else if (button === helpButton) {
+                    vscode.env.openExternal(vscode.Uri.parse(runtimeDocUrl))
```

Comment:
The intention was to link to user documentation, not runtimes. If a user needs help understanding the SAM App creation process, reading about runtimes does not help. Perhaps it is appropriate to link out to the runtimes page from the create app portion of the user docs?

Reply:
Do you have a page in mind? I don't think the SAM docs are the correct place to link people to considering that we're abstracting that away...are you thinking of linking to the toolkit docs?

Reply:
Swapping it with https://docs.aws.amazon.com/toolkit-for-vscode/latest/userguide/create-sam.html

Reply:
✔️